packages/ember-glimmer/lib/helpers/component.js

```diff
@@ -1,7 +1,143 @@
+import { CachedReference } from '../utils/references';
+import { CurlyComponentDefinition } from '../syntax/curly-component';
+import { EvaluatedArgs, EvaluatedNamedArgs, EvaluatedPositionalArgs } from 'glimmer-runtime';
+import { assert } from 'ember-metal/debug';
+import assign from 'ember-metal/assign';
+
+const CLOSURE_COMPONENT = 'ba564e81-ceda-4475-84a7-1c44f1c42c0e';
```

Comment:
@krisselden No, but it is used to stamp the definition in the constructor of `ClosureComponentDefinition` and also by `isClosureComponent`.